Makefile

```
@@ -666,6 +676,7 @@ endif
 		$$(if [ $(INSTALL_STABLE_REMOTE) = true ]; then echo '--values https://raw.githubusercontent.com/uselagoon/lagoon-charts/refs/tags/lagoon-remote-$(STABLE_REMOTE_CHART_VERSION)/charts/lagoon-remote/ci/linter-values.yaml'; else echo '--values ./charts/lagoon-remote/ci/linter-values.yaml'; fi) \
 		--set "lagoon-build-deploy.enabled=false" \
 		--set "dockerHost.registry=registry.$$($(KUBECTL) -n ingress-nginx get services ingress-nginx-controller -o jsonpath='{.status.loadBalancer.ingress[0].ip}').nip.io" \
+		--set global.rabbitMQPassword='$$($(KUBECTL) -n lagoon-core get secret lagoon-core-broker -o json | $(JQ) -r '.data.RABBITMQ_PASSWORD | @base64d')' \
```

This throws an error about missing `@base64d`:

> Error: "helm upgrade" requires 2 arguments
> 
> Usage:  helm upgrade [RELEASE] [CHART] [flags]
> /bin/sh: 11: @base64d): not found
> make[1]: *** [Makefile:664: install-lagoon-remote] Error 127

Changing the quote mechanism to be the same as done in `install-lagoon-build-deploy` target fixes it.

```suggestion
		--set "global.rabbitMQPassword=$$($(KUBECTL) -n lagoon-core get secret lagoon-core-broker -o json | $(JQ) -r '.data.RABBITMQ_PASSWORD | @base64d')" \
```